pkg/cloud/mocks/mock_ec2.go

```diff
@@ -1,144 +1,128 @@
-// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Did we use a different version of MockGen so that the content are different from the previous generated code? Which MockGen are you using? Could you add some documentation about this in [docs](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/tree/master/docs) folder

Reply:
I think it was the same version as before, but apparently it had some incompatibilities with go modules. I had to update the imports of this file manually (not good).

I updated the deps to their last minor versions and now it works fine.